_layouts/sidenav.html

```diff
@@ -15,14 +15,14 @@
   <nav aria-label="{{ site.data[page.lang].settings.accessible_labels.secondary_navigation }}" class="position-sticky pin-top">
     <ul class="usa-sidenav">
       <li class="usa-sidenav__item">
-        {% assign section_url = site.data[page.lang].settings.nav[page.sidenav].url | default: page.url %}
-        <a href="{{ section_url | prepend: site.baseurl }}" class="usa-current"
+        {% assign section_url = site.data[page.lang].settings.nav[page.sidenav].url %}
+        <a href="{{ section_url }}" class="usa-current"
```

Comment:
I notice in Federalist that links in the side navigation don't work, e.g. on "Privacy & Security" pages.

Is it expected to be fixed here?

https://federalist-17bd62cc-77b7-4687-9c62-39b462ce6fd5.app.cloud.gov/preview/18f/identity-site/LG-5864-federalist-nav-fix/policy/

Reply:
It seems like outside of the Markdown files, we should still want the `site.baseurl` prefixing? Here and for the image in `content/_landing/404.md`.

Reply:
Thanks for catching this. I added the `prepend: site.baseurl` to both the section and item urls in the privacy and security pages. 
While we're trying not to include {{ site.baseurl }} to markdown files, I noticed that the 404.md file is not included in the cms, so I think it is ok in this case.